controllers/services_controller.go

```diff
@@ -21,9 +21,10 @@ import (
 )
 
 const (
-	MetricsServiceName    = "ssp-operator-metrics"
-	OperatorName          = "ssp-operator"
-	serviceControllerName = "service-controller"
+	MetricsServiceName       = "ssp-operator-metrics"
+	OperatorName             = "ssp-operator"
+	serviceControllerName    = "service-controller"
+	appKubernetesPartOfValue = "hyperconverged-cluster"
```

Comment:
I am not sure about this value. What if SSP is not deployed by HCO? Shouldn't this value come from HCO?

Reply:
I removed it.